workers/unity/Packages/com.improbable.gdk.tools/GdkToolsConfigurationEditorExtensions.cs

```diff
@@ -28,7 +28,7 @@ public class GdkToolsConfigurationWindow : EditorWindow
 
         private readonly GUIStyle errorLayoutOption = new GUIStyle();
 
-        [MenuItem("SpatialOS/GDK tools configuration")]
+        [MenuItem("SpatialOS/GDK tools configuration", false, 201)]
```

Comment:
201 here can be moved into the the static menu priority config class too

Reply:
I recommend all `spatialos/` menu priorities to be in that class if possible. That allows us to see the layout in code easily and prevent collisions too.

Reply:
won't be possible as some menu items are in different packages. We will need such a class for every package that we have and then figure out how to make sure that the ordering is correct :(